protos/telemetry/telemetry.proto

```diff
@@ -576,7 +576,11 @@ enum FixType {
 // Battery type.
 message Battery {
     uint32 id = 3 [(mavsdk.options.default_value)="0"]; // Battery ID, for systems with multiple batteries
+    float temperature_degc = 7 [(mavsdk.options.default_value)="NaN"]; // Temperature of the battery. INT16_MAX for unknown temperature
     float voltage_v = 1 [(mavsdk.options.default_value)="NaN"]; // Voltage in volts
+    float current_battery_ma = 4 [(mavsdk.options.default_value)="NaN"]; // Battery current, -1: autopilot does not measure the current
+    float current_consumed_mah = 5 [(mavsdk.options.default_value)="NaN"]; // Consumed charge, -1: autopilot does not provide consumption estimate
```

Comment:
NAN for unknown. And what about using Ah

Reply:
I bring the units from the mavlink message, but I agree that if the end user needs to change the units every time it makes more sense that we do it right away